### DISCUSSION TITLE
How to know the status of the service camera.turn_off camere.turn_on

### DISCUSSION THREAD

**Discussion by Bellavista** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#172] [Q&A] [answered]
Sorry for the beginner question.
I am using an automation to turn the map update service on and off with camera.turn_on cameras.turn_off.
I would like to know the state the service is in, but I don't know where to start...i(sensor..switch..)
Help me!

**Comment by PiotrMachowski** [comment, marked as answer]
That is a good question! Currently there is no way to recognize if camera is turned off. I have to add it

**Reply by Bellavista** [reply]
New idea, to improve add-ons. Good job

**Reply by PiotrMachowski** [reply]
Related issue: #173
